schema/deploy/computed_columns/application_operator.sql

```diff
@@ -0,0 +1,22 @@
+-- Deploy ggircs-portal:computed_columns/application_operator to pg
+-- requires: tables/application
+
+begin;
+
+create or replace function ggircs_portal.application_operator(app ggircs_portal.application)
+returns setof ggircs_portal.organisation
+as $$
+  begin
+
+    return query (
+      select o.* from ggircs_portal.organisation o
+        join ggircs_portal.facility f on o.id = f.organisation_id
+        join ggircs_portal.application on f.id = app.facility_id
+    );
+
+  end;
+$$ language 'plpgsql' stable;
```

Comment:
can it be an `sql` function instead of `plpgslq`? AFAIK they're more efficient as postgres treats plpgsql functions as a black box

Reply:
`return query` requires plpgsql